src/sagemaker/jumpstart/notebook_utils.py

```diff
@@ -419,3 +419,16 @@ def get_model_url(
         region=region, model_id=model_id, version=model_version
     )
     return model_specs.url
+
+
+def is_valid_model_id(
+    model_id: str, script: JumpStartScriptScope = JumpStartScriptScope.INFERENCE
+) -> bool:
+    """Returns True if the model id is supported for the given script."""
```

Comment:
nit^2: `Return True if the model ID [...]`

Reply:
Agh, i missed this one